### PR TITLE
Ana/change timeline titles

### LINKED ISSUE #4812: Governance API questions and improvements
1. can depositers and voters be identity objects like proposer (name, address, image if possible)?
2. can proposer be an object instead of a string?
3. timeline and links don't need to be inside of detailedVotes
4. proposal descriptions should always include the content of the proposal. eg: `/polkadot/proposals/democracy-1` has been replaced with our "description" but the content of the parameter change is missing. let's always leave the actual proposal description in the description field and add our content as a `note` (https://kusama.polkassembly.io/referendum/81)
5. timeline should include all the steps of the governance process (including future steps - array of 3 or 4 objects) and one of the objects should be `current: true` to indicate where the proposal is in the timeline
6. can beneficiary be an object instead of a string?
7. what is the `validator` field for?
8. can we have actual IDs like polkassembly (https://kusama.polkassembly.io/referendum/81 appears as `democracy-1` for us)?
9. `percentageDepositsNeeded` is null - and why is it percentage as opposed to total number of tokens?
10. `/polkadot/proposals/democracy-1` has no date / time information at all anywhere ... is this expected?
11. `polkadot/proposals/referendum-2` all the votes (163) have the same `voter` - is `voter` supposed to be address? ideally this would be an identity object

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4812

**Description:**

I think we can safely close this issue after this change

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f6362065ad1c700e47ee72d--lunieio.netlify.app

File: api/lib/source/cosmosV0-source.js [hunk 1]
```diff
@@ -302,16 +302,16 @@ class CosmosV0API extends RESTDataSource {
           : 0,
       links,
       timeline: [
-        { title: `Proposal created`, time: proposal.submit_time },
+        { title: `Created`, time: proposal.submit_time },
         {
-          title: `Proposal deposit period ends`,
+          title: `Deposit Period Ends`,
```
Comment (faboweb) [inline api/lib/source/cosmosV0-source.js:307]:
```suggestion
          title: `Depositing Ends`,
```

File: api/lib/source/cosmosV0-source.js [hunk 2]
```diff
@@ -302,16 +302,16 @@ class CosmosV0API extends RESTDataSource {
...
           time: proposal.deposit_end_time
         },
         {
-          title: `Proposal voting period starts`,
+          title: `Voting Period Started`,
```
Comment (faboweb) [inline api/lib/source/cosmosV0-source.js:311]:
```suggestion
          title: `Voting Started`,
```

File: api/lib/source/cosmosV0-source.js [hunk 3]
```diff
@@ -302,16 +302,16 @@ class CosmosV0API extends RESTDataSource {
...
           time: proposal.voting_start_time
         },
-        { title: `Proposal voting period ends`, time: proposal.voting_end_time }
+        { title: `Voting Period Ends`, time: proposal.voting_end_time }
```
Comment (faboweb) [inline api/lib/source/cosmosV0-source.js:314]:
```suggestion
        { title: `Voting Ends`, time: proposal.voting_end_time }
```

File: api/lib/source/polkadotV0-source.js [hunk 4]
```diff
@@ -967,11 +967,11 @@ class polkadotAPI {
       timeline: [
         // warning: sometimes status.end - status.delay doesn't return the creation block. Don't know why
         {
-          title: `Proposal created`,
+          title: `Created`,
           time: proposal.creationTime
         },
         {
-          title: `Voting period opens`,
+          title: `Voting Period Started`,
```
Comment (faboweb) [inline api/lib/source/polkadotV0-source.js:974]:
```suggestion
          title: `Voting Started`,
```

File: api/lib/source/polkadotV0-source.js [hunk 5]
```diff
@@ -980,7 +980,7 @@ class polkadotAPI {
             : undefined
         },
         {
-          title: `Proposal voting period ends`,
+          title: `Voting Period Ends`,
```
Comment (faboweb) [inline api/lib/source/polkadotV0-source.js:983]:
```suggestion
          title: `Voting Ends`,
```

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:983]: It was Jordan's idea :shrug:

**Comment by jbibla** [comment]
i asked James to decide - he thinks `Period` is necessary

File: api/lib/source/cosmosV0-source.js [hunk 6]
```diff
@@ -303,23 +303,23 @@ class CosmosV0API extends RESTDataSource {
       links,
       timeline: [
         proposal.submit_time
-          ? { title: `Proposal created`, time: proposal.submit_time }
+          ? { title: `Created`, time: proposal.submit_time }
           : undefined,
         proposal.deposit_end_time
           ? {
-              title: `Proposal deposit period ends`,
+              title: `Deposit Period Ends`,
```
Comment (jbibla) [inline api/lib/source/cosmosV0-source.js:310]:
```suggestion
              title: `Deposit Period Ended`,
```

File: api/lib/source/cosmosV0-source.js [hunk 7]
```diff
@@ -303,23 +303,23 @@ class CosmosV0API extends RESTDataSource {
...
           ? {
-              title: `Proposal voting period starts`,
+              title: `Voting Period Started`,
               time: proposal.voting_start_time
             }
           : undefined,
         proposal.voting_end_time
           ? {
-              title: `Proposal voting period ends`,
+              title: `Voting Period Ends`,
```
Comment (jbibla) [inline api/lib/source/cosmosV0-source.js:322]:
```suggestion
              title: `Voting Period Ended`,
```

File: api/lib/source/polkadotV0-source.js [hunk 8]
```diff
@@ -977,19 +977,19 @@ class polkadotAPI {
...
           ? {
-              title: `Voting period opens`,
+              title: `Voting Period Started`,
               time: proposalVotingPeriodStarted
             }
           : undefined,
         proposalEndTime
           ? {
-              title: `Proposal voting period ends`,
+              title: `Voting Period Ends`,
```
Comment (jbibla) [inline api/lib/source/polkadotV0-source.js:992]:
```suggestion
              title: `Voting Period Ended`,
```

**Review by jbibla — APPROVED** [review]

**Pull request merged**